{{cookiecutter.repo_name}}/src/components/Header.js

```diff
@@ -29,7 +29,7 @@ export default (props) => (
       <Column width={12}>
         <StyledHeader>
           <Link to="/">
-            <img className="logo" src="/images/logo.svg" alt="{{ cookiecutter.project_name }}" height="auto" width={210} />
+            <img className="logo" src="/images/logo.svg" alt="{{ cookiecutter.project_name }}" height="auto" width={210} data-testid='home-image' />
```

Comment:
Only request is to use double quotes instead of single for props. (Same as below)